b3lb/rest/endpoints/b3lb/endpoints.py

```diff
@@ -51,58 +51,111 @@
     "getDefaultConfigXML"
 ]
 
+ENDPOINTS_WITH_INTERNAL_ID = [
+    "create",
+    "join",
+    "end",
+    "isMeetingRunning",
+    "getMeetingInfo",
+    "setConfigXML"
+]
+
 LEGAL_ENDPOINTS = WHITELISTED_ENDPOINTS + BLACKLISTED_ENDPOINTS
+SUPPRESSED_PARAMS = ['dialNumber', 'voiceBridge']
 
 
 ##
 # Meta Routine
 ##
+async def api_pass_through(request, endpoint="", slug=None, sub_id=0):
+    """
+    Entrypoint for BBB APIs
+    """
+    # async: workaround for @require_http_methods decorator
+    if request.method not in ["GET", "POST"]:
+        return HttpResponseNotAllowed(["GET", "POST"])
+
+    parameters = request.GET
+
+    params = {}
+    for param in parameters:
+        params[param] = parameters[param]
+
+    if "checksum" in params:
+        checksum = params["checksum"]
+        del params["checksum"]
+    else:
+        return HttpResponse("Unauthorized", status=401)
+
+    secret = await sync_to_async(lb.get_request_secret)(request, slug, sub_id)
+    if not secret:
+        return HttpResponse("Unauthorized", status=401)
+
+    if not (await lb.check_tenant(secret.secret, checksum, endpoint, request.META.get("QUERY_STRING", "")) or await lb.check_tenant(secret.secret2, checksum, endpoint, request.META.get("QUERY_STRING", ""))):
+        return HttpResponse("Unauthorized", status=401)
+
+    if endpoint in LEGAL_ENDPOINTS:
+        if endpoint in WHITELISTED_ENDPOINTS:
+            return await requested_endpoint(secret, endpoint, request, params)
+        else:
+            response = HttpResponse()
+            response.status_code = 403
+            return response
```

Comment:
Why not using [HttpResponeForbidden](https://docs.djangoproject.com/en/4.0/ref/request-response/#django.http.HttpResponseForbidden)? Are there any BBB API methods that b3lb does **not** permit? Requests not complying with the BBB API should return [HttpResponseBadRequest](https://docs.djangoproject.com/en/3.2/ref/request-response/#django.http.HttpResponseBadRequest).